packages/editor/src/yaml-table.tsx

```diff
@@ -1,295 +1,271 @@
 import {
-    VSCodeButton,
-    VSCodeCheckbox,
-    VSCodeDataGrid,
-    VSCodeDataGridCell,
-    VSCodeDataGridRow,
-    VSCodeTextField
+  VSCodeButton,
+  VSCodeCheckbox,
+  VSCodeDataGrid,
+  VSCodeDataGridCell,
+  VSCodeDataGridRow,
+  VSCodeTextField
 } from '@vscode/webview-ui-toolkit/react';
-import { useEffect, useState } from 'react';
-import { WebviewApi } from 'vscode-webview';
+import { useEffect, useState, type FormEvent } from 'react';
 import * as YAML from 'yaml';
-import './codicon.css';
+import '@vscode/codicons/dist/codicon.css';
 import './yaml-table.css';
 
-interface YAMLVariablesTableProps {
-    vscodeApi?: WebviewApi<unknown>;
-}
+type YAMLVariablesTableProps = {
+  content: string;
+  onChange: (content: string) => void;
+};
```

Comment:
This was the reason for the PR.
The integration code (in the vscode-extensions repo) should take care of this webview message connection.